build.gradle

```diff
@@ -37,7 +37,8 @@ ext {
   checkerFrameworkVersion = '3.22.1'
   checkstyleToolVersion = '8.33'
   commonsIOVersion = '2.11.0'
-  ideProbeVersion = '0.31.0'
+  ideProbeVersion = '0.31.0+9-882007b8-SNAPSHOT'
```

Comment:
Btw! sorry for asking yet another time... if you stick for `0.31.0` for this PR, will **anything** be broken? CMIIW but `probe.paths.screenshots` should work as expected?

Reply:
yes

Reply:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/5265/workflows/81de0d41-2a81-48a9-95d8-8ae1fd0b6d83/jobs/5541 
here is failed CI (with intention) run on `0.31.0+9-882007b8-SNAPSHOT`. So maybe now I remove all debug code from here and test with `0.31.0` ? or you want to test something more with `0.31.0+9-882007b8-SNAPSHOT`  ?

Reply:
Up to you, we can either merge this PR with ide-probe v0.31.0, or only merge it once we release v0.32.0, then including all v0.32.0-specific fixes (like skipping `IDEPROBE_TEST_CASE`)

Reply:
I think that wait until ide-probe release is better option (now screenshots should work because https://github.com/VirtusLab/git-machete-intellij-plugin/pull/922 so functionality is satisfied)